core/roboconf-core/src/main/java/net/roboconf/core/utils/Utils.java

```diff
@@ -361,6 +362,26 @@ public static void writeStringInto( String s, File outputFile ) throws IOExcepti
 
 
 	/**
+	 * Appends a string into a file.
+	 *
```

Comment:
When there are several parameters, it is not forbidden to add a line break between the Javadoc description and the set of parameters. But in this case, you are right. I fixed it in my working directory. I'll push it along with my next commits, as it would not be relevant to add a new commit only for this. :wink: